generator/sig_readme.tmpl

```diff
@@ -68,6 +68,18 @@ The following subprojects are owned by sig-{{.Label}}:
 {{- range .Owners }}
     - {{.}}
 {{- end }}
+{{- if .Contact.Slack }}
+  - Slack: [#{{.Contact.Slack}}](https://kubernetes.slack.com/messages/{{.Contact.Slack}})
+{{- end }}
+{{- if .Contact.MailingList }}
+  - Mailing List: [Mailing List]({{.Contact.MailingList}})
```

Comment:
```suggestion
  - [Mailing List]({{.Contact.MailingList}})
```

Reply:
Looks better and consistent with others IMO

Reply:
got it, saw this right after I pushed, squash incoming